TransceiverBase.cpp

```diff
@@ -70,7 +70,7 @@ void TransceiverBase::set (TransceiverState const& s,
             {
               do_ptt (false);
               do_post_ptt (false);
-              QThread::msleep (100); // some rigs cannot process CAT
+              QThread::msleep (PTT_DELAY_MS); // some rigs cannot process CAT
                                      // commands while switching from
```

Comment:
Actually, this already explained here in the code with a comment. Why are we adding code bloat for this? I specifically object to that. Code bloat is a common problem with open source projects "just because you can". But that doesn't make it right. As an example, on another project I work on where code runs on STM32 microcontroller chips, enough developers kept doing this type of stuff, and then it ends up having to be trimmed out because the binary wouldn't fit on the STM32 memory anymore.